src/communication/network_interface_sr.c

```diff
@@ -10890,15 +10905,25 @@ sflashback_get_summary (THREAD_ENTRY * thread_p, unsigned int rid, char *request
   /* area packing : OID list | num summary | summary info list */
   ptr = area;
 
-// *INDENT-OFF*
-  for (auto iter : context.classoid_set)
+  for (int i = 0; i < context.num_class; i++)
     {
-      ptr = or_pack_oid (ptr, &iter);
+      OID tmp_oid = classoid_queue.front ();
+      ptr = or_pack_oid (ptr, &tmp_oid);
+      classoid_queue.pop ();
     }
-// *INDENT-ON*
+
+  ptr = or_pack_int64 (ptr, start_time);
+  ptr = or_pack_int64 (ptr, end_time);
+
+  num_ptr = ptr;
 
   ptr = or_pack_int (ptr, context.num_summary);
-  ptr = flashback_pack_summary_entry (ptr, context);
+  ptr = flashback_pack_summary_entry (ptr, context, &tmp_num);
+
+  if (context.num_summary != tmp_num)
+    {
+      or_pack_int (num_ptr, tmp_num);
```

Comment:
1. I can't find the client-side considering this new packed one.
2. Do `context.num_summary` has to be packed? `tmp_num` seems to be the one reliable.

Reply:
As we discussed offline, num_summary is overwritten by tmp_num